server/types/app.go

```diff
@@ -51,6 +52,9 @@ type (
 
 		// RegisterTendermintService registers the gRPC Query service for tendermint queries.
 		RegisterTendermintService(clientCtx client.Context)
+
+		// Return the multistore instance
+		CommitMultiStore() sdk.CommitMultiStore
```

Comment:
Even if it's only "internal" API breaking, can we still add a changelog entry?

Reply:
moved the changelog entry to API breaking section.